src/clock.rs

```diff
@@ -3,7 +3,7 @@ use std::time::{Duration, Instant as StdInstant};
 
 /// When using a virtual clock, this defines how many nanoseconds we pretend are passing for each
 /// basic block.
-const NANOSECONDS_PER_BASIC_BLOCK: u64 = 10;
+const NANOSECONDS_PER_BASIC_BLOCK: u64 = 5000;
```

Comment:
All that this PR does is change a magic number. There's no particular reason to expect that basic blocks are uniform in size, or that the interpreter or typical host CPUs won't get faster or slower. Are we just expecting to tweak this number from time to time?

Reply:
I don't think the number itself matters that much. The order of magnitude does. I'll leave a comment about the magicness of it.

If we have reason to tune it again in the future, we should find a better metric.